packages/flutter/lib/src/widgets/statistics_overlay.dart

```diff
@@ -36,6 +36,7 @@ enum StatisticsOption {
   visualizeEngineStatistics,
 }
 
+/// Displays performance statistics.
```

Comment:
Yeah, we should have a page on the web site that shows what the different parts mean and how to diagnose issues using it.  I'll add a comment to that effect.